gutenTAG/base_oscillations/random_walk.py

```diff
@@ -6,11 +6,15 @@
 
 from . import BaseOscillation
 from .interface import BaseOscillationInterface
-from ..utils.default_values import default_values
```

Comment:
just to be sure: Is it ok to remove this import?

Reply:
I don't recall removing it. Should maybe be answered by @CodeLionX

Reply:
Ok, thanks for clarifying. I am fine with this as is.